src/test/kotlin/no/nav/omsorgspenger/koronaoverføringer/rivers/MockLøsninger.kt

```diff
@@ -55,12 +60,28 @@ private fun JsonMessage.leggTilLøsningPåHentePersonopplysninger(
 )
 
 private fun JsonMessage.leggTilLøsningPåHenteOmsorgspengerSaksnummer(
-    fra: Identitetsnummer, til: Identitetsnummer) = leggTilLøsning(
+    fra: Identitetsnummer, til: Identitetsnummer
+) = leggTilLøsning(
     behov = HentOmsorgspengerSaksnummerMelding.HentOmsorgspengerSaksnummer,
     løsning = mapOf(
         "saksnummer" to mapOf(
             fra to "foo",
             til to "bar"
         )
     )
-)
\ No newline at end of file
+)
+
+private fun JsonMessage.leggTilLøsningPåVurderRelasjoner(
+    til: Set<Identitetsnummer>
+) = leggTilLøsning(
+    behov = VurderRelasjonerMelding.VurderRelasjoner,
+    løsning = mapOf(
+        "relasjoner" to til.map {
+            mapOf(
+                "relasjon" to "INGEN",
+                "identitetsnummer" to it,
+                "borSammen" to false
```

Comment:
Ettersom `borSammen` alltid er false så burde egentlig testen som allerede er der feile.
For nå skal den jo da filtrere bort alle barn, og med 0 barn har du 0 tilgjengelige dager